back/app/services/projects_finder_service.rb

```diff
@@ -4,8 +4,7 @@ def initialize(projects, user = nil, params = {})
     @user = user
     @page_size = (params.dig(:page, :size) || 500).to_i
     @page_number = (params.dig(:page, :number) || 1).to_i
-    @finished = params[:finished]
-    @archived = params[:archived]
+    @filter_by = params[:filter_by]
```

Comment:
Making the parameter we are interested in explicit here, to make following which param is used where easier than if we use @params = params

Reply:
I'm not too fussed either way, tbh. I prefer it this way, but yes it doesn't matter much while the service is this simple.